.cp2k/Makefile

```diff
@@ -452,38 +340,38 @@ FYPPFLAGS ?= -n
 	$(CC) -c $(CFLAGS) $<
 
 # Compile the CUDA/HIP files
-ifneq ($(NVCC),)
```

Comment:
This means that the name of this variable will have to be changed in CP2K's Makefile and arch files accordingly ...

Reply:
So, take a look at my PR https://github.com/cp2k/cp2k/pull/798
The idea is to use:

https://github.com/cp2k/cp2k/pull/798/files#diff-1f791d555349bad7410338a8fad208ee

I don't really want to change CP2K documentation, so we can keep NVCC in CP2K and ACC in DBCSR. For people they what to try HIP in CP2K (note, it is not documented) they can still use it by setting `NVCC` and `NVFLAGS` (or setting in the arch file and use `override`).

Reply:
Please let me know if my solution is fine with you...

Reply:
Yes, this looks good to me!